src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -147,11 +147,38 @@ the output document.
 ![Image placeholder replaced by the image data](../images/image_tag_final.png)
 
 ## Tables
+### Creating Tables dynamically using HTML
 
-Rows of a table can be dynamically populated by repeating over an array
-of objects from the input data.
+Text tags having HTML Table string as an input data can be used to insert tables in the output document.
 
-**How to insert Tables**
+
+Below is a json containing textTag with an HTML Table value.
+
+```json
+{
+  "Table_Data": "<table border='1' cellpadding='0' cellspacing='0' align='right' style='width:30%'>\n\t<tr>\n\t\t<th>Name</th>\n\t\t<th>Age</th>\n\t\t<th>Gender</th>\n\t</tr>\n\t<tr>\n\t\t<td>Sheldon</td>\n\t\t<td>29</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Leonard</td>\n\t\t<td>30</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Howard</td>\n\t\t<td>28</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Raj</td>\n\t\t<td>29</td>\n\t\t<td>Male</td>\n\t</tr>\n</table>\n"
+}
+```
+**Usage**
+
+Placeholder variable get replaced by a table generated from input html table string.
```

Comment:
Replace with a slightly more explanation like 'Placeholder tag is replaced by the table generated using html table string provided in the placeholder tag value'.

Reply:
Done.